rpyc/core/stream.py

```diff
@@ -7,7 +7,14 @@
 import socket
 import errno
 from rpyc.lib import safe_import, Timeout, socket_backoff_connect
-from rpyc.lib.compat import poll, select_error, BYTES_LITERAL, get_exc_errno, maxint  # noqa: F401
+from rpyc.lib.compat import (
+    poll,
+    select_error,
+    BYTES_LITERAL,
+    get_exc_errno,
+    maxint,
+    unix_socket,
+)
```

Comment:
what happened to `# noqa: F401` (what does it mean anyway?)

Reply:
lost in patching process. I'll put it back. It's a hint to some linters not to flag the line

Reply:
`noqa: F401` is for no warning on unused imports